File: Chapter6/f.cpp
#include <iostream>

using namespace std;

string spaceRemover(string s) {
    string res = "";
    for (int i = 0; i < s.length(); i++) {
        if (s[i] != ' ' && s[i] != '_') {
            res += s[i];
        }
    }
    return res;
}

int main() {
    string s;
    getline(cin, s);

    cout << spaceRemover(s) << '\n';
    return 0;
}

File: Chapter6/f.pas
var
    s: String;

function space_remover(s: String): String;
    var 
        res: String;
        i: Integer;
    Begin
        res := '';
        for i:= 1 to length(s) do
            if not((s[i] = ' ') or (s[i] = '_')) then
                res := res + s[i];
        space_remover := res;
    End;

Begin
    readln(s);
    writeln(space_remover(s));
End.
